docs/victoriametrics-cloud/get-started/overview.md

```diff
@@ -0,0 +1,14 @@
+---
+weight: 1
+title: VictoriaMetrics Cloud Overview
+menu:
+  docs:
+    parent: get-started
```

Comment:
do we really want to have additional sub menu?

Reply:
I think that's the idea, and I'm fine with it. I struggled quite a bit to see how we can both add it without being repetitive, and this is the best I came up with. It's actually good IMO, because now when you click on "VictoriaMetrics Cloud" section, you get the overview, and it's also under the Get started section, without writing extra superfluous content. It's also done this way in other products.

I'll tidy up the get started menu page once I have the content.